kratos/includes/kratos_components.h

```diff
@@ -32,6 +32,24 @@
 
 namespace Kratos
 {
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+    
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
```

Comment:
Do we really need these?

Reply:
It doesn't make any harm

Reply:
And it's part of the documentation

Reply:
But it's a lot of code that you have to scroll away. And the documentation doesn't work anyways...

Reply:
I usually also remove them, makes files much smaller...